app/containers/MessageActions.js

```diff
@@ -155,7 +155,7 @@ export default class MessageActions extends React.Component {
 
 	getPermalink = async(message) => {
 		try {
-			return await RocketChat.getPermalink(message);
+			return await RocketChat.getPermalink(message, 'message');
```

Comment:
in folder `app/constants` create file `permalinkMessageTypes.js` and move strings `'message'`, `'room'` there.